src/sentry/integrations/mixins/issues.py

```diff
@@ -121,10 +121,14 @@ def get_group_link(self, group, **kwargs):
 
     def get_group_description(self, group, event, **kwargs):
         output = self.get_group_link(group, **kwargs)
-        if isinstance(event, GroupEvent) and event.occurrence is not None:
+        if (
+            event
+            and isinstance(event, GroupEvent)
+            and event.occurrence is not None
+            and group.issue_category == GroupCategory.FEEDBACK
+        ):
             body = ""
```

Comment:
```suggestion
```

Reply:
Flake8 gives me `undefined name 'body'` :(

Reply:
oh that's so weird